crates/stackable-operator/src/cluster_resources.rs

```diff
@@ -32,12 +32,12 @@ use crate::{
     client::{Client, GetApi},
     commons::{
         cluster_operation::ClusterOperation,
-        listener::Listener,
         resources::{
             ComputeResource, LIMIT_REQUEST_RATIO_CPU, LIMIT_REQUEST_RATIO_MEMORY,
             ResourceRequirementsExt, ResourceRequirementsType,
         },
     },
+    crd::listener::v1alpha1 as listener_v1alpha1,
```

Comment:
We could keep it in line with the others by importing the module:

```suggestion
    crd::listener,
```

Then referring to things like `listener::v1alpha1::Listener`

Reply:
Done in 5f4030206716404f5469f9f46ea49050d8a3bb47.